sympl/_components/netcdf.py

```diff
@@ -81,6 +87,12 @@ def store(self, state):
                 cache_state = {name: state[name] for name in name_list}
             else:
                 cache_state = state.copy()
+
+            # replace cached variable names with their aliases
+            for oldname, newname in self._aliases.items():
+                if oldname in cache_state.keys():
+                    cache_state[newname] = cache_state.pop(oldname)
```

Comment:
you also need to raise an exception if `'sympl_name'] in cache_state[newname].attrs` (probably a `ValueError` with a description of what's wrong, and then do `cache_state[newname].attrs['sympl_name'] = oldname`.